### DISCUSSION TITLE
Proposed Changes to Deep Argo Flags

### DISCUSSION THREAD

**Discussion by nzilberman** [OneArgo/ADMT#90] [General]
Dear all,

The Deep Argo Mission Team met last week to discuss Deep Argo flag modifications that would increase Deep Argo data use. The proposed plan is summarized in the PowerPoint attached to this discussion. Please use github to let us know if you would have any objection to the proposal. We would need to receive your comments no later than January 24th.

Nathalie and Virginie

[DeepArgo_flag_cpcor_change_proposal.pptx](https://github.com/user-attachments/files/18101367/DeepArgo_flag_cpcor_change_proposal.pptx)

**Comment by fionacarse** [comment]
Great news - the assimilators will be able to use the data in ocean forecasts when it is flagged "1". I am happy :0)

**Reply by nzilberman** [reply]
Great! Thank you Fiona for the support!
